xml/api/pacemakerd-health-2.25.rng

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<grammar xmlns="http://relaxng.org/ns/structure/1.0"
+         datatypeLibrary="http://www.w3.org/2001/XMLSchema-datatypes">
+
+    <start>
+        <ref name="element-pacemakerd-health"/>
+    </start>
+
+    <define name="element-pacemakerd-health">
+        <choice>
+            <element name="pacemakerd">
+                <optional>
+                    <attribute name="sys_from"> <text /> </attribute>
+                </optional>
+                <attribute name="state"> <text /> </attribute>
+                <optional>
+                    <attribute name="last_updated"> <text /> </attribute>
+                </optional>
+            </element>
+
+            <!-- @COMPAT: The below is deprecated since 2.1.5 -->
+            <element name="pacemakerd">
+                <attribute name="state"> <text /> </attribute>
+                <attribute name="last_updated"> <text /> </attribute>
+            </element>
```

Comment:
The second pacemakerd element isn't necessary. The first covers this case as well. (Going from mandatory to optional is backward compatible, as older output will still validate.)

Reply:
Yep, forgot to update this after s/pacemakerd-health/pacemakerd/